documentation/src/docs/recovery-procedures/metadata-table-segment-recovery.md

```diff
@@ -0,0 +1,118 @@
+<!--
+Copyright Pravega Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+# Metadata Table Segment Recovery
```

Comment:
Shouldn't this be `Metadata Table Segment Attribute Index Recovery`? Because what we can recover is the index, not the main segment, right?

Reply:
corrected.